### DISCUSSION TITLE
Improve POI and favorites panel usage

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#675] [Q&A] [answered]
When pressing on a POI or favorite panel, it is completely unclear which POI or favorite is currently selected. When pressing a POI or favorite button, the respective POI should be centered, its size doubled and displayed in a different color so that it can be distinguished from others. Also the current "in drive direction" should be kept and not switch "north on the top". This continues until another is selected or the seconds have elapsed to return to the normal display.

example 
![grafik](https://github.com/user-attachments/assets/9801ca62-0c77-4ab4-a8e6-28e71facac93)

I have pressed on the first, the name is displayed and pronounced, but it is impossible to see while driving which it was.

**Comment by devemux86** [comment]
Changing overlays appearance while navigation is active is not something we want to do for various (technical) reasons.

> the respective POI should be centered

This already happens. The map centers on the POI of the tapped panel.

If you display a lot of POIs (not recommended) and at low zooms, there to be a lot of them.

**Reply by devemux86** [reply]
All markers (favorites, pois, etc.) can already change size and color in app settings.

What we can do extra here is to zoom at a large level when tapping a nearby panel,
so the map more clearly shows the tapped nearby poi / favorite (it already centers).

**Reply by DolomitesTom** [reply]
Maybe let's try... where can the size and color of POI categories be changed?

**Reply by devemux86** [reply]
- Size: "Settings | Map | Appearance | Icon size (markers)"

- Color: "Settings | Search | Points of interest color"

**Comment by devemux86** [comment, marked as answer]
> zoom at a large level when tapping a nearby panel

- Available in https://github.com/devemux86/cruiser/discussions/684
